server/libs/request.js

```diff
@@ -17,12 +20,25 @@ const getHttpsClient = () => {
 
 function createClient({ baseURL, auth }) {
 
+    const logger = getLogger();
+
     const client = axios.create({
         baseURL,
         auth,
         httpsAgent: isProduction ? getHttpsClient() : null
     });
 
+    //Retry 3 times with 5 second gaps
+    axiosRetry(client, { retries: backendRequestRetries, retryDelay: () => { return 5000; } });
+
+    //Warn about retries
+    client.interceptors.request.use(request => {
+        if('axios-retry' in request){
+            logger.warn('REQUESTRETRY', { url: request.url, retryCount: request['axios-retry'].retryCount });
```

Comment:
Should this log the reason for the retry? Or the correlation ID of the initiating request?